bin/seedUserStreaks.ts

```diff
@@ -156,9 +158,12 @@ const computeReadingStreaksData = async (
     insertCount += ids.length;
   }, QUEUE_CONCURRENCY);
 
-  let nextUser = undefined;
+  const lastSeed = await getRedisObject(LAST_SEED_KEY);
+  let nextUser = lastSeed ? JSON.parse(lastSeed) : undefined;
+
   while (nextUser !== null) {
     nextUser = await addNextBatch(con, insertQueue, nextUser);
+    await setRedisObject(LAST_SEED_KEY, JSON.stringify(nextUser));
```

Comment:
Just make sure this key gets removed when you are done. Also you might want to log it just in case as well since redis can get cleared at any point.